signature/policy_config.go

```diff
@@ -33,6 +42,30 @@ func (err InvalidPolicyFormatError) Error() string {
 
 // FIXME: NewDefaultPolicy, from default file (or environment if trusted?)
 
+// DefaultPolicy returns the default policy of the system.
+// Most applications should be using this method to get the policy configured
+// by the system administrator.
+// ctx should usually be nil, can be set to override the default.
+// NOTE: When this function returns an error, either immediately abort,
+// or use the returned *Policy (which will never be nil); this policy
+// fails closed and refuses to trust any image.
+func DefaultPolicy(ctx *types.SystemContext) (*Policy, error) {
+	policyPath := defaultPolicyPath(ctx)
+	policy, err := NewPolicyFromFile(policyPath)
```

Comment:
we can directly use `defaultPolicyPath(ctx)` here but not a blocker - feel free to ignore as it may impact readability